lib/actions/model.js

```diff
@@ -61,4 +61,26 @@ module.exports = {
       return xhr;
     };
   },
+
+  validate(brainstemKey, modelId, attributes, options = {}) {
+    const {
+      validateOptions = {},
+      preValidateAction,
+      postValidateAction,
+      trackKey,
+    } = options;
+
+    const storageManager = StorageManager.get();
+
+    return (dispatch) => {
+      if (preValidateAction) dispatch(preValidateAction);
+
+      const Model = storageManager.storage(brainstemKey).model;
+      const validationErrors = new Model({ id: modelId }).validate(attributes, validateOptions);
```

Comment:
To avoid creating a new instance of a model everytime we want to validate, how about if we kept a cached copy of a bare bones model and just invoked method? We could do that by up top (outside of this method definition) doing

`let cachedModel = new Backbone.Model()`

then here having

```
cachedModel.clear({silent: true});
const validationErrors = Model.prototype.validate.call(cachedModel, Object.assign({ id: modelId }, attributes), validateOptions);
```

It's a little more complicated, but could help with memory.